.agents/automation/reports/dependency-maintenance.md

```
@@ -7,33 +7,39 @@ Conservative dependency maintenance audit and updates to resolve version-drift a
 CHANGES_APPLIED
 
 ## Scope Inspected
-- Root configuration and package manifests (`package.json`, `pnpm-lock.yaml`)
-- Dashboard dashboard SPA app package manifest (`apps/web/app/package.json`)
+- Root lockfile (`pnpm-lock.yaml`)
+- Dashboard SPA app package manifest (`apps/web/app/package.json`)
 - Platform admin SPA package manifest (`apps/web/admin/package.json`)
+- Marketing Astro package manifest (`apps/web/marketing/package.json`)
 - Gradle dependencies config (`gradle/libs.versions.toml`)
 
 ## Changes Applied
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)